docs/how-delta-lake-works/architecture-of-delta-table.md

```diff
@@ -86,7 +86,7 @@ Create another pandas DataFrame and append it to the Delta table to see how this
 
 ```python
 df = pd.DataFrame({"num": [8, 9], "letter": ["dd", "ee"]})
-write_deltalake(f"{cwd}/tmp/delta-table", df, mode="append")
+write_deltalake("tmp/some-table", df, mode="ovewrite")
```

Comment:
The snippet of text preceding this mentions appending the dataframe to the table, I am assuming the change in `mode` here is in error?

Reply:
Correct, I got to a bit to zealous, this one should be `append`.